dev_guide/executing_remote_commands.adoc

```diff
@@ -14,6 +14,13 @@ toc::[]
 You can use the CLI to execute remote commands in a container. This allows you
 to run general Linux commands for routine operations in the container.
 
+[IMPORTANT]
+====
+The `oc exec` command does not work when accessing privileged containers. In
```

Comment:
Maybe start this with "For security purposes," and possibly make it link to the related security errata? https://access.redhat.com/errata/RHSA-2015:1650